install/kubernetes/cilium/templates/cilium-agent/clusterrole.yaml

```diff
@@ -1,6 +1,6 @@
-{{- if and (.Values.agent) (not .Values.preflight.enabled) }}
+{{- if and .Values.agent (not .Values.preflight.enabled) }}
 {{- /*
-Keep file in synced with cilium-preflight-clusterrole.yaml
+Keep file in synced with cilium-preflight/clusterrole.yaml
```

Comment:
It's old word, but I update it as your suggestion :+1:

Reply:
I see you did another commit just for that. I just meant updating the line in the same commit since you were already changing `-` to `/`, not to create a separate commit :) But OK keep it like this, and thanks for fixing the typo!